lib/MusicBrainz/Server/Entity/URL/eonkyo.pm

```diff
@@ -0,0 +1,23 @@
+package MusicBrainz::Server::Entity::URL::eonkyo;
```

Comment:
To follow the previous suggestion, this file should be renamed accordingly, and its first line be changed too:
```suggestion
package MusicBrainz::Server::Entity::URL::EOnkyo;
```

Reply:
Also resolved in [666952b](https://github.com/metabrainz/musicbrainz-server/pull/3328/commits/666952b9614532302c2914ed7fb51e96d0d79d12)!